domain_access/src/DomainAccessManager.php

```diff
@@ -92,7 +92,7 @@ public static function getAccessValues(FieldableEntityInterface $entity, $field_
    * {@inheritdoc}
    */
   public static function getAllValue(FieldableEntityInterface $entity) {
-    return $entity->hasField(DOMAIN_ACCESS_ALL_FIELD) ? $entity->get(DOMAIN_ACCESS_ALL_FIELD)->value : NULL;
+    return $entity->hasField(DOMAIN_ACCESS_ALL_FIELD) ? (bool) $entity->get(DOMAIN_ACCESS_ALL_FIELD)->value : NULL;
```

Comment:
This is the only non-test related change. It seems Drupal 9 added a strict check to assertTrue/assertFalse.

```  /**
   * Get the all affiliates field values from an entity.
   *
   * @param \Drupal\Core\Entity\FieldableEntityInterface $entity
   *   The entity to retrieve field data from.
   *
   * @return bool
   *   Returns TRUE if the entity is sent to all affiliates.
   */
  public static function getAllValue(FieldableEntityInterface $entity);
```

The return value is documented as a boolean but it seems this method is returning integers.